c-cannons/bangazon-initial_site
# Search functionality

#### Descriptions

Added fuzzy search functionality to search for products by both name and city.

#### Steps to Test

```
git fetch origin search-functionality
git checkout search-functionality
```

Using the search bar and toggle:
1. Search for products when toggle is on (blue)
2. Search for products when toggle is off (grey)

**Note: make sure to test whether you can search from any page and not just the home page.**

#### Link to Feature Ticket
#5 
#4
